python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -180,6 +184,60 @@ def replace_illegal_columns_chars(char: str, database: BaseDatabase) -> str:
 
         return normalize_config
 
+    def get_openlineage_facets(self) -> OpenLineageFacets:
+        """
+        Returns the lineage data
+        """
+        input_files = resolve_file_path_pattern(
+            self.input_file.path,
+            self.input_file.conn_id,
+            normalize_config={},
+            filetype=self.input_file.type.name,
+        )
```

Comment:
Why do we need to resolve it here, a comment explaining it would be good

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1025/commits/7dad9a6fc4f0bcc5205f25fe51fb0b3f629a621e)